### PR TITLE
Initial inclusion of an ID card for GOCDB

### PR DESCRIPTION
**Pull request by gwarf**
Initial inclusion of an ID card for GOCDB.
It's from https://wiki.egi.eu/wiki/GOCDB

Content, naming and presentation to be reviewed/agreed.

Screenshot 2021-02-18 at 18 30 57

### CODE REVIEW

**Comment by gwarf** [comment]
@paolini78 I had a first test at adding an identity card, we need to discuss what is meaningful to be presented there.

**Comment by enolfc** [comment]
Should we have one of these for every service?

**Comment by gwarf** [comment]
> Should we have one of these for every service?

Yes it's the intent. It's to replace things we have in the wiki for the central services, like https://wiki.egi.eu/wiki/GOCDB

File: content/en/internal/configuration-database/_index.md [hunk 1]
```diff
@@ -9,9 +9,36 @@ type: "docs"
...
+| URL                          | https://goc.egi.eu                                                    |
+| Support Email                | gocdb-admin@mailman.egi.eu                                            |
+| Helpdesk Support Unit        | EGI Services and Service components -> Configuration Database (GOCDB) |
+| Configuration Database entry | https://goc.egi.eu/portal/index.php?Page_Type=Site&id=335             |
+| Supplier                     | [UKRI](https://www.ukri.org/)                                         |
+| Roadmap                      | [Roadmap](https://wiki.egi.eu/wiki/GOCDB/Roadmap)                     |
+| Release notes                | [Release notes](https://github.com/GOCDB/gocdb/releases)              |
+| Issue tracker for developers | https://github.com/GOCDB/gocdb/issues                                 |
+| Source code                  | https://github.com/GOCDB                                              |
+| License                      | gLite/Apache 2                                                        |
```
Comment (enolfc) [inline content/en/internal/configuration-database/_index.md:41]: github repo says Apache 2

File: content/en/internal/configuration-database/_index.md [hunk 2]
```diff
@@ -9,9 +9,36 @@ type: "docs"
...
+| Privacy Policy               | https://wiki.egi.eu/wiki/GOCDB/data_privacy                           |
```
Comment (enolfc) [inline content/en/internal/configuration-database/_index.md:42]: What about this? https://goc.egi.eu/privacy.html

File: content/en/internal/configuration-database/_index.md [hunk 3]
```diff
@@ -9,9 +9,36 @@ type: "docs"
...
+
+<!-- markdownlint-disable line-length no-bare-urls -->
+
+| Property                     | Value                                                                 |
+| ---------------------------- | --------------------------------------------------------------------- |
+| Name                         | Configuration Database                                                |
+| Description                  | Central registry of the infrastructure topology                       |
+| URL                          | https://goc.egi.eu                                                    |
+| Support Email                | gocdb-admin@mailman.egi.eu                                            |
+| Helpdesk Support Unit        | EGI Services and Service components -> Configuration Database (GOCDB) |
```
Comment (enolfc) [inline content/en/internal/configuration-database/_index.md:34]: Do we have a link for this?

**Review by enolfc — COMMENTED** [review]
Some minor things

File: content/en/internal/configuration-database/_index.md (hunk 1 above)
Comment (gwarf) [reply content/en/internal/configuration-database/_index.md:41]: Updated

File: content/en/internal/configuration-database/_index.md (hunk 2 above)
Comment (gwarf) [reply content/en/internal/configuration-database/_index.md:42]: Updated

File: content/en/internal/configuration-database/_index.md (hunk 3 above)
Comment (gwarf) [reply content/en/internal/configuration-database/_index.md:34]: If we had some basic stuff about using GGUS under https://docs.egi.eu/support/ we could point to that.
I don't think (may be wrong, never saw it but never tried) we can easily make an URL allowing to create a ticket for a specific SU.

Comment (gwarf) [reply content/en/internal/configuration-database/_index.md]: I've added a link to the Helpdesk page (that is a bit rough)

**Comment by gwarf** [comment]
> Some minor things

Update screenshot addressing comments.
Screenshot 2021-03-29 at 14 21 58

**Review by enolfc — APPROVED** [review]
LGTM!

**Pull request merged**